net/haproxy/src/opnsense/service/templates/OPNsense/HAProxy/haproxy.conf

```diff
@@ -441,7 +441,7 @@
 {# ############################### #}
 
 global
-    uid                         80
+    #uid                         80
```

Comment:
Please don't do this. HAProxy should NOT run as root in the default configuration.
Please add a new Option "Run as root" to "General Settings", it should be an "advanced" option and the default value set to disabled.